skimage/transform/radon_transform.py

```diff
@@ -124,6 +124,64 @@ def _sinogram_circle_to_square(sinogram):
     return np.pad(sinogram, pad_width, mode='constant', constant_values=0)
 
 
+def iradon_filter(filter_name, size):
+    """
+    Create iradon filter.
+
+    Generate a reconstruction kernel to be used in iradon during
+    the filtered back-projection algorithm.
+
+    Parameters
+    ----------
+    filter_name : str
+        Name of the filter.
+    size : int
+        Length of the filter array.
+
+    Returns
+    -------
+    kernel : array_like, dtype=float
+        Reconstruction filter to be used in iradon.
+
+    References
+    ----------
+    .. [1] AC Kak, M Slaney, "Principles of Computerized Tomographic
+           Imaging", IEEE Press 1988.
+    """
+    # Construct the Fourier filter
+    n1 = np.arange(0, size / 2 + 1, dtype=np.int)
+    n2 = np.arange(size / 2 - 1, 0, -1, dtype=np.int)
+    n = np.concatenate((n1, n2))
+    f = np.zeros(size)
+    f[0] = 0.25
+    f[1::2] = -1 / (np.pi * n[1::2])**2
+
+    omega = 2 * np.pi * fftfreq(size)
+    kernel = 2 * np.real(fft(f))         # ramp filter
+    if filter_name == "ramp":
+        pass
+    elif filter_name == "shepp-logan":
+        # Start from first element to avoid divide by zero
+        kernel[1:] *= np.sin(omega[1:] / 2) / (omega[1:] / 2)
```

Comment:
Should be [`sinc`](https://docs.scipy.org/doc/numpy/reference/generated/numpy.sinc.html).

Since `sinc` is defined as sin(pi x) / (pi x), I suggest using `f` instead of omega. and doing
```
f = fftfreq(size)
kernel *= np.sinc(f)
```